buf-kvutils.yaml

```diff
@@ -1,39 +0,0 @@
-# Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
-# SPDX-License-Identifier: Apache-2.0
-
-version: v1beta1
-
-build:
-  roots:
-    - daml-lf/archive/src/main/protobuf
```

Comment:
JFYI @remyhaemmerle-da, this directory is no longer tested.

Reply:
:bow: